web/app.js

```diff
@@ -1581,6 +1581,9 @@ const PDFViewerApplication = {
     ) {
       console.warn("Warning: Interactive form support is not enabled");
       this._delayedFallback(UNSUPPORTED_FEATURES.forms);
+    } else if (info.AreSignaturesPresent) {
```

Comment:
Shouldn't this always be done when signatures are present, since as-is the warning might not actually display otherwise in some cases?